### PR TITLE
apps/protobuf: add serialize/deserialize tests for protocol buffer

### PR DESCRIPTION
**Pull request by pillip8282**
Added more test cases for protocol buffer to check the result of serialize/deserialize by data types.

### CODE REVIEW

File: apps/examples/protobuf_enc/Makefile [hunk 1]
```diff
@@ -0,0 +1,152 @@
+###########################################################################
+#
+# Copyright 2017 Samsung Electronics All Rights Reserved.
```
Comment (sunghan-chang) [inline apps/examples/protobuf_enc/Makefile]: 2018

**Comment by seinfra** [comment]
Target : [ea5cdb68e4a0bb9b1129f2f23fefed1197d02381] - Code Rule Check (C++) OK.

File: apps/examples/protobuf_enc/Makefile (hunk 1 above)
Comment (pillip8282) [reply apps/examples/protobuf_enc/Makefile]: Thanks I'll fix

**Comment by seinfra** [comment]
Target : [a01fa0f861f44d550ec596f229837eddf80cf5d3] - Code Rule Check (C++) OK.

**Pull request merged**